docs/en/connector/sink/Assert.md

```diff
@@ -0,0 +1,105 @@
+# Assert
+
+> # Sink plugin: Assert [Flink]
+
+## Description
+
+A flink sink plugin which can assert illegal data by user defined rules
+
+:::tip
+
+Engine Supported and plugin name
+
+* [ ] Spark
+* [x] Flink: AssertSink
+
+:::
+
+## Options
+
+| name                    | type     | required | default value |
+| ----------------------- | -------- | -------- | ------------- |
+| rules         		  | ConfigList | yes      | -             |
+| &ensp;field_name        | `String` | yes       | -     |
+| &ensp;field_type        | `String` | no       | -          |
+| &ensp;field_value | ConfigList | no       | -             |
+| &ensp;&ensp;rule_type         | `String`    | no       | -             |
+| &ensp;&ensp;rule_value         | double    | no       | -             |
+
+
+### rules
```

Comment:
Please add type after option name. Like other connector doc

Reply:
sorry, I don't understand this suggestion. The type is just after the name now.

Reply:
```suggestion
### rules [string]
```